occurrences/event_api_services.py

```diff
@@ -105,3 +119,36 @@ def get_event_time_range_from_occurrences(p_event: Optional["PalvelutarjotinEven
     if first_occurrence and last_occurrence:
         return first_occurrence.start_time, last_occurrence.end_time
     return None, None
+
+
+def get_enrollable_event_time_range_from_occurrences(
+    p_event: Optional["PalvelutarjotinEvent"],
+):
+    if not p_event or not p_event.enrolment_start:
+        """
+        1. No p_event set yet
+        2. 2nd step of the UI's form wizard, that includes the occurrences
+        and enrolment data, is not yet full filled.
+        3. The event is externally enrollable
+        4. The event is not enrollable at all
+        """
+        return None, None
+
+    occurrences = p_event.occurrences.filter(cancelled=False).order_by("end_time")
+    end_time = occurrences.last().start_time
```

Comment:
This raises if there aren't any occurrences, but I suppose we can assume there always are at this point?